src/libcore/array.rs

```diff
@@ -264,6 +267,269 @@ macro_rules! array_impls {
     }
 }
 
+#[cfg(not(bootstrap))]
+mod impls_using_const_generics {
```

Comment:
These are separate because I don't want to rely on the bootstrap compiler's const generics.